tests/conftest.py

```diff
@@ -241,7 +241,7 @@ def assert_gas_values_match_for_call(
                     gas_limit=self.data[key].limit,
                 )
 
-        def assert_gas_costs_match(self, key, gas_cost, *, abs_delta=0):
+        def assert_gas_costs_match(self, key, gas_cost, *, abs_delta=100):
```

Comment:
I would strongly advice against setting the delta to 100. 
The gas costs should usually be predictable afaik, so the only reason I am aware of are flaky tests, So I would rather set this in the flaky tests.

Reply:
I could not find a reason for the variation of gas usage from the tests and their fixtures, so I would not say so.